frontend/src/lib/components/RadioOption/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './index.scss'
+
+export interface RadioOption {
+    key: string
+    label: string
+    icon?: any // any, because Ant Design icons are some weird ForwardRefExoticComponent type
+}
+
+interface RadioOptionProps {
+    options: RadioOption[]
+    selectedOption?: null | string
```

Comment:
Let's make this mandatory - undefined is weird here, this is a controlled component.